WebRoot/js/zimbraMail/abook/model/ZmContactList.js

```diff
@@ -160,10 +160,39 @@ function(callback, errorCallback, accountName) {
 	DBG.println(AjxDebug.DBG1, "loading contacts from " + params.restUri);
 	appCtxt.getAppController().sendRequest(params);
 
+	// wait for Contacts package to be loaded
+	AjxDispatcher.addPackageLoadFunction("Contacts", new AjxCallback(this, this._loadSharingFolders));
+
 	ZmContactList.addDlFolder();
 	
 };
 
+ZmContactList.prototype._loadSharingFolders =
```

Comment:
I think _loadSharedFolders would me more meaningful. what you say?

Reply:
how about this?
- _searchSharingFolders
- _getContactsInSharingFolders  or _fetchContactsInSharingFolders

Reply:
changed to `_loadSharedFolders`